app/src/main/java/org/simple/clinic/enterotp/EnterOtpEffectHandler.kt

```diff
@@ -42,6 +42,7 @@ class EnterOtpEffectHandler @AssistedInject constructor(
         .addTransformer(RequestLoginOtp::class.java, activateUser())
         .addAction(ShowSmsSentMessage::class.java, uiActions::showSmsSentMessage, schedulers.ui())
         .addConsumer(FailedLoginOtpAttempt::class.java, { bruteForceProtection.incrementFailedOtpAttempt() }, schedulers.io())
+        .addAction(ShowNetworkError::class.java, { uiActions.showNetworkError() }, schedulers.ui())
```

Comment:
This is a personal preference, I like using method references instead of full lambdas for this sort of thing.

```kotlin
.addAction(ShowNetworkError::class.java, uiActions::showNetworkError, schedulers.ui())
```